mobile-app/app/components/TransactionCard.tsx

```diff
@@ -122,6 +123,7 @@ function SetAmountButton({
 }: SetAmountButtonProps): JSX.Element {
   const decimalPlace = 8;
   let value = amount.toFixed(decimalPlace);
+  const [userPressedState, setUserPressedState] = useState(false);
```

Comment:
Can we use simple variable name? like `isPressed`

Reply:
updated variable name to `const [isPressed, setIsPressed] = useState(false);`